wfsim/core/s2.py

```diff
@@ -139,18 +155,19 @@ def get_electron_yield(n_electron, positions, z_obs, config, resource):
         # Average drift time of the electrons
         drift_time_mean, drift_time_spread = S2.get_s2_drift_time_params(z_obs, positions, config, resource)
 
-        # Absorb electrons during the drift
+        # Absorb electrons during the drift, clipping to avoid artefacts of maps
+        drift_time_mean=np.clip(drift_time_mean, a_min=0, a_max=np.inf)
```

Comment:
Can you just change in the function from `_drift_time_mean` to `drift_time_mean` and avoid the clipping here? I am not sure now why I included it, I do not think such a situation can show up (or at least I forgot if it did), but in this way we do not have too much stuff out of the function